index.html

```diff
@@ -8,43 +8,220 @@
     <title>Selcuk Guler's Personal Portfolio website</title>
   </head>
   <body>
-    <main>
-      <div id="topbar">
-        <div id="appbar">
+    <div id="appbar">
           <div id="timer-cont">
             <div id="timer-text">9:41</div>
           </div>
           <div id="statusbar">
           </div>
-        </div>
-        <div class="container" id="toolbar">
+    </div>
+    <header class="container" id="toolbar">
           <div id="logo-placeholder"></div>
           <nav id="hamburger-cont">
             <div id="hamburger-icon"></div>
           </nav>
-        </div>
-      </div>
+    </header>
+    <main>
       <section id="home">
-        <div id="greeting-container">
-          <header class="greetings">
-            <span class="black-text">Hey there.<br>I'm Selçuk.</span>
-            <br>
-            <span class="green-text">I'm a software<br>developer</span>
-          </header>
-        </div>
+        <h1 id="greeting-container" class="greetings">
+          <span class="black-text">Hey there.<br>I'm Selçuk.</span>
+          <br>
+          <span class="green-text">I'm a software<br>developer</span>
+        </h1>        
         <div id="message-cont">
           <p class="black-text" id="support-txt"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
             don’t hesitate to contact me.</p>
         </div>
-        <ul id="sm-icons-cont">
+        <ul class="sm-icons-cont">
             <li id="twitter"><a href="#"><span class="sm-icons"></span></a></li>
             <li id="linkedin"><a href="#"><span class="sm-icons"></span></a></li>
             <li id="medium"><a href="#"><span class="sm-icons"></span></a></li>
             <li id="github"><a href="#"><span class="sm-icons"></span></a></li>
             <li id="angellist"><a href="#"><span class="sm-icons"></span></a></li>
         </ul>
         <div id="home-bg-img"></div>
+        <div id="action-btn-cont">
+          <div id="action-btn"></div>
+        </div>
+      </section>
+      <section id="works">
+        <div id="works-head-cont">
+          <h2 class="black-text section-header" id="works-header">My Recent Works</h2>
+          <div id="indicator"></div>
+        </div>
+        <div id="cards-cont">
+          <div class="card" id="card1">
+            <div class="project-info" id="project1">
+              <h3 class="project-name">Multi-Post Stories Gain+Glory</h3>
+              <ul class="pr-tags-ul">
+                <li class="pr-tag">Ruby on Rails</li>
+                <li class="pr-tag">CSS</li>
+                <li class="pr-tag">JavaScript</li>
+                <li class="pr-tag">HTML</li>
+              </ul>
```

Comment:
- [ ] [OPTIONAL] While this current `color` looks good and without much impact on user experience, the expected color for the text is `color: #3a4a42` not `#000`. Apart from matching the design a little closer, I think it's more aesthetically impressive :+1: